packages/mdc-textfield/bottom-line/README.md

```diff
@@ -0,0 +1,57 @@
+<!--docs:
```

Comment:
Do we want these subcomponent docs to show up on material.io? IIRC that's what this front-matter is used for.